### PR TITLE
Style edge finance

### PR DESCRIPTION
**Pull request by tlssuse**
@chabowski : Document has been updated for the new TRD stylesheet and Rancher branding.

### CODE REVIEW

File: kubernetes/start/suse/adoc/gs_rancher_edge-analytics_finance_stocks.adoc [hunk 1]
```diff
@@ -1,21 +1,85 @@
...
+// - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
+
+
+// - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
+// authors
+:author1_firstname: Navin
+:author1_surname: Chandra
+:author1_jobtitle: Google Summer of Code 2022 Contributor
+:author1_orgname: openSUSE
+// :author1_email: abhinavsharma332@gmail.com
```
Comment (navin772) [inline kubernetes/start/suse/adoc/gs_rancher_edge-analytics_finance_stocks.adoc:62]: Correct Email to - navinchandra772@gmail.com

**Review by navin772 — CHANGES_REQUESTED** [review]
Correct Email ID

File: kubernetes/start/suse/adoc/gs_rancher_edge-analytics_finance_stocks.adoc (hunk 1 above)
Comment (tlssuse) [reply kubernetes/start/suse/adoc/gs_rancher_edge-analytics_finance_stocks.adoc:62]: Thanks for catching this, Navin.

**Review by chabowski — APPROVED** [review]
Thank you - changes are approved.

**Pull request merged**